modules/gdscript/gdscript_parser.h

```diff
@@ -476,29 +472,34 @@ class GDScriptParser {
 		DataType return_type;
 		virtual DataType get_datatype() const { return return_type; }
 		virtual void set_datatype(const DataType &p_datatype) { return_type = p_datatype; }
-		CastNode() { type = TYPE_CAST; }
+
+		CastNode() {
+			type = TYPE_CAST;
+		}
 	};
 
 	struct AssertNode : public Node {
-		Node *condition;
-		Node *message;
-		AssertNode() :
-				condition(0),
-				message(0) {
+		Node *condition = 0;
+		Node *message = 0;
```

Comment:
Those should be `nullptr`.

Reply:
Good catch. I had those in my current `git stash` as I plan to PR fixing to missing `nullptr` usage, but it's best to solve it now for those two.